rllib/utils/replay_buffers/multi_agent_episode_replay_buffer.py

```diff
@@ -0,0 +1,671 @@
+import copy
+from collections import defaultdict
+import numpy as np
+import scipy
+from typing import Dict, List, Optional, Tuple, Union
+
+from ray.rllib.core.columns import Columns
+from ray.rllib.env.multi_agent_episode import MultiAgentEpisode
+from ray.rllib.utils.replay_buffers.episode_replay_buffer import EpisodeReplayBuffer
+from ray.rllib.utils import force_list
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.spaces.space_utils import batch
+from ray.rllib.utils.typing import SampleBatchType
+
+
+class MultiAgentEpisodeReplayBuffer(EpisodeReplayBuffer):
+    def __init__(
```

Comment:
docstring

Reply:
Will come as soon as we keep the architecture as is.